hw/ip_templates/rv_plic/fpv/vip/rv_plic_assert_fpv.sv.tpl

```diff
@@ -104,4 +105,7 @@ module ${module_instance_name}_assert_fpv #(parameter int NumSrc = 1,
   // but smaller than the threshold
   `ASSERT(IdChangeWithIrq_A, !$stable(irq_id_o[tgt_sel]) && irq_id_o[tgt_sel] != 0 |->
           irq_o[tgt_sel] || ((irq_id_o[tgt_sel]) == $past(i_high_prio) && !$past(irq)))
+
+  // When fatal alert happens then only reset can clear it.
+  `ASSERT(FatalAlertNeverLow_A, ##1 !$fell(fatal_alert_i))
```

Comment:
A genuine question: is the `##1` actually needed here? Is the point that we get `$fell` on the first cycle after reset if the signal is constant zero?

I've just looked at the 1800 spec and I don't *think* I can see that, but it's possible I've missed something...

Reply:
We just talked about that but I am still leaving a note here; the point to add ##1 delay is for the property. Initially, fatal_alert_i should be X and X->0 transition is also a $fell(true) criteria. So, the assertion fails on the 2nd cycle.